src/testdir/test_popup.vim

```diff
@@ -2067,69 +2085,78 @@ func Test_pum_maxwidth_multibyte()
   call VerifyScreenDump(buf, 'Test_pum_maxwidth_08', {'rows': 8})
   call term_sendkeys(buf, "\<ESC>")
 
-  call StopVimInTerminal(buf)
-endfunc
-
-func Test_pum_maxwidth_with_many_items()
```

Comment:
Keep the test `Test_pum_maxwidth_with_many_items()` that was added in the following commit as it is.
If the results change due to this PR, replace `src/testdir/dumps/Test_pum_maxwidth_with_many_items_*.dump`.
(The value of pummaxwidth may need to be adjusted)
https://github.com/vim/vim/commit/f13c8561544dad4f82b7f4f71041d35f55b5feaa

Reply:
Emm there is no need for more situations to be covered by the current test. Extra tests would not have a significant effect here.

Reply:
It looks like you just want to erase other people's tracks.